dbos/queues_test.go

```diff
@@ -30,14 +30,16 @@ This suite tests
 */
 
 func queueWorkflow(ctx DBOSContext, input string) (string, error) {
-	step1, err := RunAsStep(ctx, queueStep, input)
+	step1, err := RunAsStep[string](ctx, func(context context.Context) (string, error) {
```

Comment:
So do we still have to pass in `[string]` input type here? Or can it be inferred?

Reply:
nope this is because I forgot to remove one of the generic in the `RunAsStep` signature -- I clearly wasn't paying enough attention. Sorry about that.